udata/frontend/helpers.py

```diff
@@ -164,12 +164,12 @@ def owner_url(obj):
 @front.app_template_filter()
 @contextfilter
 def avatar(ctx, user, size, classes=''):
-    markup = '''
-        <a class="avatar {classes}" href="{url}" title="{title}">
-            <img src="{avatar_url}" class="avatar" alt="{title}"
-            width="{size}" height="{size}"/>
-        </a>
-    '''.format(
+    markup = ''.join((
```

Comment:
I liked the multiline string better but YMMV.

Reply:
Yep me too but the extras spaces where the source of the visual breakage :/